### PR TITLE
Rename upgrade to upgrade_downgrade

### PR DESCRIPTION
**Pull request by carolynhu**
Rename the folder name since we have 3 test scenarios: `upgrade`, `downgrade`, `upgrade-downgrade`

### CODE REVIEW

**Comment by carolynhu** [comment]
https://github.com/istio/test-infra/pull/2882

File: crossgrade/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# How to run the upgrade test
+# How to run the crossgrade test
```
Comment (brian-avery) [inline crossgrade/README.md:1]: First time I've heard the term crossgrade. Can we please define it?

Comment (carolynhu) [reply crossgrade/README.md:1]: yeah, I'm not sure which word to use here. Basically I want to include the meaning for both upgrade and downgrade. Maybe just update it to upgrade and downgrade?

Comment (carolynhu) [reply crossgrade/README.md:1]: Offline discussed with @brian-avery, we think `upgrade_downgrade` is clearer than defining a new term in the README here.

**Pull request merged**